src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -202,19 +231,21 @@ def create_api(self):
         self.log.info("Successfully created API")
         return api_id
 
-    def create_resource(self, parent_id=""):
+    def create_or_update_resource(self, parent_id=""):
         """Create the specified resource.
 
         Args:
             parent_id (str): The resource ID of the parent resource in API Gateway
         """
         resource_name = self.trigger_settings.get('resource', '')
         resource_name = resource_name.replace("/", "")
-        created_resource = self.client.create_resource(restApiId=self.api_id,
-                                                       parentId=parent_id,
-                                                       pathPart=resource_name)
-        resource_id = created_resource['id']
-        self.log.info("Successfully created resource")
+        try:
+            created_resource = self.client.create_resource(restApiId=self.api_id,
+                                                           parentId=parent_id,
+                                                           pathPart=resource_name)
+            resource_id = created_resource['id']
+            self.log.info("Successfully created resource")
+        except:
```

Comment:
This try/except was unintended to be in the commit. Bouncing back and forth with production issues.